src/index.js

```diff
@@ -1,31 +1,40 @@
 import * as babel from '@babel/core';
 import { createFilter } from 'rollup-pluginutils';
-import createPreflightCheck from './preflightCheck.js';
-import helperPlugin from './helperPlugin.js';
-import { addBabelPlugin, escapeRegExpCharacters, warnOnce } from './utils.js';
-import { RUNTIME, EXTERNAL, HELPERS } from './constants.js';
+import preflightCheck from './preflightCheck.js';
+import bundledHelpersPlugin from './bundledHelpersPlugin.js';
+import { addBabelPlugin, escapeRegExpCharacters } from './utils.js';
+import { RUNTIME, EXTERNAL, BUNDLED, INLINE, HELPERS } from './constants.js';
 
 const unpackOptions = ({
 	extensions = babel.DEFAULT_EXTENSIONS,
+	skipBabelHelpersCheck = false,
 	// rollup uses sourcemap, babel uses sourceMaps
 	// just normalize them here so people don't have to worry about it
 	sourcemap = true,
 	sourcemaps = true,
 	sourceMap = true,
 	sourceMaps = true,
 	...rest
-} = {}) => ({
-	extensions,
-	plugins: [],
-	sourceMaps: sourcemap && sourcemaps && sourceMap && sourceMaps,
-	...rest,
-	caller: {
-		name: 'rollup-plugin-babel',
-		supportsStaticESM: true,
-		supportsDynamicImport: true,
-		...rest.caller,
-	},
-});
+} = {}) => {
+	if (!rest.babelHelpers) {
```

Comment:
Let's default this to "bundled", but if it's not given explicitly then we should warn users that they should configure this manually and that we have chosen bundled for them. The same thing happens regarding "mode" in webpack since v4.

I would also move that wall of text to README.md and just link to appropriate heading from the warning.